mynewprogram.txt

```diff
@@ -0,0 +1,3 @@
+a = 1
+b = 2
+print(a + b)
```

Comment:
Noice! So the variable a and b are added together and their result is printed. I'll allow it. In the future, make sure to have more descriptive variables though!